libraries/primitive_consul_health.rb

```diff
@@ -30,7 +30,7 @@ def are_checks_passing?(tries)
         checks = Diplomat::Check.checks
 
         non_passing = @options[:checkids]
-          .select { |id| checks[id]['CheckID'] }
+          .map { |id| "service:#{id}" }
```

Comment:
From the idscussion I had with you @kamaradclimber It was not clear that we will use other checks then the service one.
As it seems that it could be I will change the commit to just update the doc to have it more clear (at least for me ;) ) indicating that the check id in checkids is the composition from the check type (service) and the id of the check (For ex.: service:myhealthcheck)